### PR TITLE
Tweaks to "browse by audience" of Resources homepage

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
- Fixes the URL for the audience tag
- Adds back the beta banner
- Adds forgotten GraphQL attribute `fieldAudienceRsHomepage`

## Testing done
Visually confirmed changes using http://localhost:3001/preview?nodeId=8296

## Screenshots
image

## Acceptance criteria
- [ ] "Browse by audience" section is correct

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/drupal/graphql/taxonomy-fragments/GetTaxonomies.graphql.js [hunk 1]
```diff
@@ -7,13 +7,15 @@ module.exports = `
           path
         }
         name
+        fieldAudienceRsHomepage
```
Comment (kelsonic) [inline src/site/stages/build/drupal/graphql/taxonomy-fragments/GetTaxonomies.graphql.js:10]: This is a bool? 🙂

Comment (kelsonic) [reply src/site/stages/build/drupal/graphql/taxonomy-fragments/GetTaxonomies.graphql.js:10]: Would this be used at the template layer or in metalsmith's plugins?

**Review by kelsonic — APPROVED** [review]
LGTM! 🎉 Added a question

File: src/site/stages/build/drupal/graphql/taxonomy-fragments/GetTaxonomies.graphql.js (hunk 1 above)
Comment (ncksllvn) [reply src/site/stages/build/drupal/graphql/taxonomy-fragments/GetTaxonomies.graphql.js:10]: Yeah `true` or `null` from what I am seeing

**Review by timwright12 — APPROVED** [review]

**Pull request merged**